spec/requests/medications_spec.rb

```diff
@@ -52,17 +57,17 @@
       before { sign_in user }
 
       context 'when the medication exists' do
-        medication.save!
         it 'returns a successful response' do
+          medication.save!
           get medication_path(medication)
           expect(response).to be_successful
         end
       end
 
       context 'when the medication does not exist' do
-        it 'redirects to the medications page'
+        it 'redirects to the medications page' do
           get medication_path(medication.id + 1)
-          expect(response).to redirect_to(medications_path)
+          expect(response).to have_http_status(302)
```

Comment:
is one of these preferable to the other? Seems to me that `redirect_to` tests output more thoroughly where it is possible to use it.

Reply:
I think if I were reading a test and it says `redirects to the medication page`, I'd probably expect it to use `redirect_to` whereas if the test reads `redirects the user` then I'd just use a generic 302.

Reading rspec tests is kind of like reading a sentence in English so I'd probably expect the test to do what it reads to do.